docs/user/accumulo/internals.rst

```diff
@@ -1,24 +0,0 @@
-Internals
-=========
-
-Indexing Strategies
```

Comment:
Did this discussion go away?

Reply:
Yeah... it was out of date, and seemed deeper in the weeds than I'd guess most people would go. Seems like it would be more appropriate in the developers guide, if we ever wrote one.